### ISSUE TITLE
IJulia fixes

### ISSUE THREAD

**Issue by denizyuret** [denizyuret/Knet.jl#97]
Add convolutional net to the softmax IJulia.
Fix RNN to use the more efficient indexing approach in rnn IJulia and charlm example.

**Comment by denizyuret** [comment]
rnnlm shows that column major with input/output merge/split is more efficient in high vocab.

**Comment by denizyuret** [comment]
- examples/overfitting/softmax.ipynb still missing convolutional net (could also add ladder networks etc to show learning efficiency).
- most rnn examples now use the new cudnn interface, check the notebooks.

**Comment by denizyuret** [comment]
Display iframe/html/javascript.
Allow user to input a hand-drawn image.

https://github.com/JuliaGizmos/Interact.jl 
https://discourse.julialang.org/t/custom-widget-in-ijulia-interact-jl/3578
